src/components/containers/partitioned_vector/partitioned_vector_component_double.cpp

```diff
@@ -0,0 +1,30 @@
+//  Copyright (c) 2017 Hartmut Kaiser
```

Comment:
Does it make sense to add all built-in and std types where possible?

Reply:
We can do that eventually. I wanted to be sure first that the compilation times for our tests are really reduced, so for now only those types needed by the tests are pre-compiled.

Reply:
Shouldn't we at least document which types we already registered? Otherwise, users will either end up with strange linker errors or ODR problems.

Reply:
Sure, this needs to be done. Currently we don't have any documentation for `partitioned_vector` ;)